docs/src/experiments/spherical_delaunay_stereographic.jl

```diff
@@ -0,0 +1,168 @@
+#=
+# Spherical Delaunay triangulation using stereographic projections
+
+This is the approach which d3-geo-voronoi and friends use.  The alternative is STRIPACK which computes in 3D on the sphere.
+The 3D approach is basically that the 3d convex hull of a set of points on the sphere is its Delaunay triangulation.
+=#
+
+import GeometryOps as GO, GeoInterface as GI
+import Proj # for easy stereographic projection - TODO implement in Julia
+import DelaunayTriangulation as DelTri # Delaunay triangulation on the 2d plane
+import CoordinateTransformations, Rotations
+
+using Downloads # does what it says on the tin
+using JSON3 # to load data
+using CairoMakie # for plotting
+import Makie: Point3d
+
+points = Point3{Float64}.(JSON3.read(read(Downloads.download("https://gist.githubusercontent.com/Fil/6bc12c535edc3602813a6ef2d1c73891/raw/3ae88bf307e740ddc020303ea95d7d2ecdec0d19/points.json"), String)))
+
+pivot_ind = findfirst(isfinite, points)
+pivot_point = points[pivot_ind]
+necessary_rotation = Rotations.RotY(pi) * Rotations.RotY(-pivot_point[2]) * Rotations.RotZ(-pivot_point[1])
+
+net_transformation_to_corrected_cartesian = CoordinateTransformations.LinearMap(necessary_rotation) ∘ UnitCartesianFromGeographic()
+
+stereographic_points = (StereographicFromCartesian() ∘ net_transformation_to_corrected_cartesian).(points)
+
+# Iterate through the points and find infinite/invalid points
+max_radius = 1
+really_far_idxs = Int[]
+for (i, point) in enumerate(stereographic_points)
+    m = hypot(point[1], point[2])
+    if !isfinite(m) || m > 1e32
+        push!(really_far_idxs, i)
+    elseif m > max_radius
+        max_radius = m
+    end
+end
+far_value = 1e6 * sqrt(max_radius)
+if !isempty(really_far_idxs)
+    stereographic_points[really_far_idxs] .= Point2(far_value, 0)
+end
+
+# Add infinite horizon points
+triangulation_points = copy(stereographic_points)
+push!(triangulation_points, Point2(0, far_value))
+push!(triangulation_points, Point2(-far_value, 0))
+push!(triangulation_points, Point2(0, -far_value))
+
+triangulation = DelTri.triangulate(triangulation_points)
+
+triangulation.points[1:end-3] .= stereographic_points
+triangulation.points[end-2:end] .= (stereographic_points[pivot_ind],)
```

Comment:
If you want to stick the API, you could also here use `get_points(triangulation)` and work with that result instead of using `.points` which I don't think I've put as the API (not intentionally anyway).

Reply:
I'll probably plot the triangles with faces from the triangulation and precomputed coordinates, this was more of an experiment (which failed, unfortunately)